docs/scaling/DEVELOPMENT.md

```diff
@@ -205,4 +205,4 @@ rules to take the Activator back out of the serving path.
 
 _Currently the Slow Brain is not implemented and the desired concurrency level
 is hardcoded at 1.0
-([code](https://github.com/knative/serving/blob/7f1385cb88ca660378f8afcc78ad4bfcddd83c47/cmd/autoscaler/main.go#L36))._
+([code](https://github.com/knative/serving/blob/master/cmd/autoscaler/main.go#L51))._
```

Comment:
This is not the correct value that's been referred to here. I think it should be:

https://github.com/knative/serving/blob/master/config/config-autoscaler.yaml#L33

Reply:
Update: Should be

https://github.com/knative/serving/blob/01787a8cd70eab441d28872621bc26ad061bf14b/config/config-autoscaler.yaml#L47